lib/active_admin/resource/attributes.rb

```diff
@@ -37,7 +37,10 @@ def sti_col?(c)
       end
 
       def counter_cache_col?(c)
-        c.name.end_with?("_count")
+        counter_cache_columns = resource_class.reflect_on_all_associations(:has_many)
+                                              .select(&:has_cached_counter?)
+                                              .map(&:counter_cache_column)
+        counter_cache_columns.include?(c.name)
```

Comment:
Seeing where this method is called as ultimately part of a loop, should we memoize the value?